Changes

TheUnholyWrath opened the pull request:
Making a bunch of terrible changes so I can ask who broke test. Uploading WIP so I can add the things I actually want to change on the sly

- Blinks are back on the menu. Darko said they crashed on cast and they don't do that anymore
- <s>Battlefury is completely OP</s>
- Yasha and Kaya buffed (who cares about consistency really)
- Correct Adaptive Strike scaling
- Nothing else

lovesmuffins commented on game/scripts/npc/items/blinks/item_arcane_blink.txt:
```diff
@@ -72,7 +72,7 @@
       "duration"    "6"
       "cast_pct_improvement"    "35 40 45 50 55" //OAA
       "manacost_reduction"    "25 30 35 40 45"
-      "debuff_amp"          "20 23 26 29 32"
```
y not tho

TheUnholyWrath replied:
scaling on scaling is stupid. if arcane blink is weak it can be changed other ways

lovesmuffins replied:
id remove manacost or stop that scaling. its the more ueselss option

lovesmuffins commented on game/scripts/npc/items/item_bfury.txt:
```diff
@@ -60,7 +60,7 @@
       {
         "var_type"                                        "FIELD_FLOAT"
-        "bonus_health_regen"                              "7.5 11 14 17 20"
+        "bonus_health_regen"                              "0" //OAA
```
y tho

TheUnholyWrath replied:
hp regen does nothing on this item, bfury is too expensive imo, giving melee heroes some love. likely imba but better direction imo

DarkoniusXNG replied:
We can argue that regen is useless on some heroes, but I offer a better solution above and probably more imba solution.

lovesmuffins commented on game/scripts/npc/items/item_yasha_and_kaya.txt:
```diff
@@ -34,9 +34,9 @@
       "bonus_intellect"     "16 24 32 48 72"
       "bonus_attack_speed"  "12 17 22 27 32"
       "mana_regen_multiplier"   "50 60 70 80 90"
-      "movement_speed_percent_bonus"  "8 10 12 14 16" //OAA
```
y tho

TheUnholyWrath replied:
sny and snk are way stronger items, ynk gets the worst of both worlds with 2 nerfs based on other items. if consistency matters then I'll revert but otherwise ynk is garbage and needs help, isn't even close to enough but a start

DarkoniusXNG replied:
We like consistency but YnK is the worst item in the game so these changes are fine.

lovesmuffins replied:
> We like consistency but YnK is the worst item in the game so these changes are fine.

i disagree personally its just that a lot of the heroes who could benefit from YnK have other options. If we were big-brain like Icefrog we'd nerf the other items to increase ynk being bought.

lovesmuffins replied:
(also tier 5 core = trident ty)

DarkoniusXNG commented on game/scripts/npc/abilities/crystal_maiden_freezing_field.txt:
```diff
@@ -34,7 +34,7 @@
+      "bonus_armor"                                       "20 25 30 35 40"  //OAA
       "explosion_interval"                                "0.1"
       "movespeed_slow"                                    "-40 -50 -60 -70 -80" //OAA
       "attack_slow"                                       "-60 -90 -120 -150 -180" //OAA
```
Nerf this, -10 difference every level

DarkoniusXNG commented on game/scripts/npc/abilities/crystal_maiden_freezing_field.txt:
```diff
@@ -43,7 +43,7 @@
       "damage" //OAA
       {
-        "value"                                           "110 250 390 810 1510"
+        "value"                                           "110 220 330 660 1320"
```
Put 105/225/345/705/1305.

DarkoniusXNG commented on game/scripts/npc/abilities/dazzle_shallow_grave.txt:
```diff
@@ -26,7 +26,7 @@
     // Time
     //-------------------------------------------------------------------------------------------------------------
-    "AbilityCooldown"                                     "24 22 20 18 17 16" //OAA
+    "AbilityCooldown"                                     "24 22 20 18 18 18" //OAA
```
I calculated that with Octarine Core, Aghanim Scepter and Spell Prism and if Dazzle spams every spell off cd he can get Permanent Grave. Even if Dazzle can't use Poison Touch and scepter ability consistently he can achieve this (there is 0.88s time of no Grave). So I would nerf this to 24 seconds at all levels.

DarkoniusXNG commented on game/scripts/npc/abilities/sandking_epicenter.txt:
```diff
@@ -46,7 +46,7 @@
         "LinkedSpecialBonus"                              "special_bonus_unique_sand_king"
       }
-      "epicenter_damage"                                  "100 200 300 600 900" //OAA
+      "epicenter_damage"                                  "100 180 260 520 780" //OAA
```
I would put 500/740 for OAA levels.

DarkoniusXNG commented on game/scripts/npc/items/custom/item_trumps_fists.txt:
```diff
@@ -68,7 +68,7 @@
       {
         "var_type"                                        "FIELD_INTEGER"
-        "bonus_all_stats"                                 "55 75"
+        "bonus_all_stats"                                 "40 50"
```
Adjust the stats to be the same as Skadi. Don't make the item shit.

TheUnholyWrath replied:
Judecca is way easier to apply to multiple heroes and is non committal. It should give even less stats than what I gave it but it builds out of skadi. The zeus v bat thing is a really weird thing to bring up, kinda conveniently ignoring backline applying judecca killing me in larger fights by hiding behind a massively losing 1v1 matchup for zeus

TheUnholyWrath replied:
Anyway I'll revert the damage changes and shelve it for a bit

DarkoniusXNG commented on game/scripts/npc/items/custom/item_trumps_fists.txt:
```diff
@@ -83,12 +83,12 @@
       {
         "var_type"                                        "FIELD_INTEGER"
-        "heal_prevent_percent"                            "75"
+        "heal_prevent_percent"                            "60 70"
```
I had Judecca on Zeus the other day. I wasn't doing any damage to your Batrider. Nerf it more and you wouldn't need any magic resistance to sustain the damage.

DarkoniusXNG commented on game/scripts/npc/items/item_bfury.txt:
```diff
@@ -13,7 +13,7 @@
     "ItemRequirements" //OAA
     {
-      "01"                                                "item_broadsword;item_claymore;item_pers;item_quelling_blade"
+      "01"                                                "item_broadsword;item_claymore;item_void_stone;item_quelling_blade"
```
Instead of removing the stat that you hate, you can just remove Claymore from the recipe. Sounds like a better option to me.

DarkoniusXNG commented on game/scripts/npc/items/item_bloodthorn.txt:
```diff
@@ -65,14 +65,14 @@
     "AbilityValues"
     {
-      "bonus_intellect"                                   "20 30 45 65 90"
+      "bonus_intellect"                                   "20 25 30 35 40"
```
Why make INT scaling shit? I am not a fan. Put 20/25/35/50/70.

TheUnholyWrath replied:
seemed really weird considering how much else bloodthorn gives now. its not really an int item anymore and nothing like this scaled anywhere near as well.

DarkoniusXNG commented on game/scripts/npc/items/item_bloodthorn.txt:
```diff
@@ -65,14 +65,14 @@
       "silence_duration"                                  "5"
       "silence_damage_percent"                            "30"
-      "target_crit_multiplier"                            "130 135 145 160 180"
+      "target_crit_multiplier"                            "130 140 150 160 170"
```
I am not sure if this is a good idea. This is going back to the values when Bloodthorn was S tier item.

TheUnholyWrath replied:
I changed it for the greatest of all reasons, it looks really stupid.

DarkoniusXNG commented on game/scripts/npc/items/item_revenants_brooch.txt:
```diff
@@ -65,14 +65,14 @@
       "bonus_intellect"   "45 50 60 75 95"
       "bonus_attack_speed"  "40 45 50 55 60"  
-      "bonus_armor"     "8 9 11 14 18"
+      "bonus_armor"     "8 9 10 11 12"
```
I am not a fan of the bad armor scaling. If you want to nerf it, then nerf the first lvl as well while preserving the scaling. Like this: 6/7/9/12/16 (Witch Blade gives 6 armor)

lovesmuffins commented on game/scripts/npc/abilities/morphling_adaptive_strike_agi.txt:
```diff
@@ -41,7 +41,7 @@
       {
         "var_type"                                        "FIELD_FLOAT"
-        "damage_base"                                     "40 60 80 120 240 480" //OAA
+        "damage_base"                                     "0 40 80 120 240 480" //OAA
```
ADAPTIVE STRIKE IS NOT GOOD

lovesmuffins replied:
AGILITY THAT IS

lovesmuffins replied:
this has got to be one of the oddest nerfs I've seen.

TheUnholyWrath replied:
I played a round of morph and he just did too much damage, Darkonius and I agreed this was the best way of dealing with how powerful adaptive strike is levels 1 and 2

lovesmuffins commented on game/scripts/npc/abilities/invoker_sun_strike.txt:
```diff
@@ -27,9 +27,9 @@
+      "damage" //OAA
       {
-        "value"         "100 162.5 225 287.5 350 412.5 475 537.5 950 1140 1330 1520 1710 1900 2090"
+        "value"         "100 180 260 340 420 500 580 660 820 980 1140 1300 1460 1620 1880"
```
?????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????

TheUnholyWrath replied:
Damge

lovesmuffins commented on game/scripts/npc/abilities/spectre_dispersion.txt:
```diff
@@ -38,13 +38,13 @@
       {
         "var_type"                                        "FIELD_INTEGER"
-        "shard_dagger_damage"                             "400" //OAA
+        "shard_dagger_damage"                             "300"
```
🤔

TheUnholyWrath replied:
Fuck spectre honestly shard_dagger_damage is too high to begin with, Darkonius was pretty stupid to make a change like this to an already imba hero

DarkoniusXNG replied:
This is the damage threshold when spectral dagger procs, so the lower the value the better. Bad name for kv so it's confusing.

lovesmuffins commented:
could you like go through and explain your process to literally every change cause none of them make sense. ty :)

TheUnholyWrath commented:
Just for you loves this is the list of everything I have changed so far and why:
- Freezing field does too much damage, gives too much armor. Darkonius said to nerf the attack speed slow as well since its imba
- Dazzle is insane rn, good juju is a problem so I wanted to see if setting it at 1.5s helped solve that
- Black hole shard pierces bkb, baby nerf
- Invoker spells have been rescaled. I'm not finished with him yet but Chaos Meteor, Deafening Blast and Cataclysm are his bread and butter in doing insane amounts of damage. Damage at later levels is worse but pre 25 is better. I think Deafening Blast disarm is insane so I nerfed it, Darko disagrees with me
- Fuck Medusa
- Adaptive Strike scaled wrong, it was 40/60/80/120 as a counting error on my end. Its gonna go to 60/80/100/120
- Sand King ult does too much damage
- Each blink kinda has 1 thing that already scales and therefore doesn't need to. Debuff duration is also an incredibly toxic thing to have in the game. When I first made blinks I think Darkonius said something about it but we were nerfing the cast point reduction so I overlooked it. Cast point reduction has since been buffed so now I think debuff duration scaling is stupid.
- Theres this midgame timing where you go from kinda tanky to mysteriously dying and its always judecca. The item really bugs me but Darkonius insists that its a useless dead item and nerfing it further is insane. I wanted to scale the effect as well, but making the caster more killable maybe does something. Not sure how many people agree with me/have noticed the judecca timing but Darkonius definitely doesn't 
- I thought I reverted all the bfury changes whoops lmao. HP regen seems so core to the item that it rubs Darko the wrong way. We spent a while trying to come up with a compromise but I can't be convinced that HP regen is a good use of gold to build towards a battle fury. Making it cheaper is nice but the main goal was to remove a distinctly dota aspect from the item that translates pretty badly into OAA. I was kinda blind to how stupid a Damage + Mana Regen bfury ends up looking though.
- Bloodthorn gives too much int for something thats not an int item. The crit damage values looked stupid. I don't like ranged heroes.
- Revenants Brooch is expensive and gives a lot of decently valuable stats. I'm still trying to figure out where the right position for the item is. Its so weird. 18 seemed like too much armor though.
- Skadi is a high value item that can be picked up pretty much every game. Giving it less stats maybe creates more build variety.
- Mostly reverting Wraith Pact buffs. The cd is still way better than vanilla, Damage reduction probably doesn't need to scale.
- Yasha and Kaya is super awkward and stuck between SnY and KnS. I hope valve does something with it because the stats it gives are pretty trash on their own. Having to build around a mediocre item sucks.
